src/main/java/com/programmers/ticketparis/controller/member/SellerController.java

```diff
@@ -0,0 +1,44 @@
+package com.programmers.ticketparis.controller.member;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.programmers.ticketparis.dto.ApiResponse;
+import com.programmers.ticketparis.dto.member.SellerCreateRequest;
+import com.programmers.ticketparis.dto.member.SellerResponse;
+import com.programmers.ticketparis.service.member.SellerService;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/api/sellers")
+@RequiredArgsConstructor
+public class SellerController {
+
+    private final SellerService sellerService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public ApiResponse<Long> createAccount(@Valid @RequestBody SellerCreateRequest sellerCreateRequest,
+        HttpServletRequest httpServletRequest
+    ) {
+        Long sellerId = sellerService.createAccount(sellerCreateRequest);
+
+        return ApiResponse.of(httpServletRequest.getRequestURI(), sellerId);
+    }
+
+    @GetMapping("/{sellerId}")
+    public ApiResponse<SellerResponse> findSellerById(@PathVariable("sellerId") Long sellerId,
```

Comment:
`p2`

`@PathVariable("sellerId)`의 경우 안에 인자로 들어가는 값과 매개변수가 동일한 경우 `@PathVariable`로 인자를 기입하지 않고 사용이 가능합니다! 👍👍 조금 더 간결하게 변경해보시는 건 어떨까요?! 

```suggestion
    public ApiResponse<SellerResponse> findSellerById(@PathVariable Long sellerId,
```

Reply:
오 인자를 없애는 편이 더 가독성이 좋네요!! 바로 반영했습니다. 감사합니다~~
6101cce8394852a966cee4aa995d9d39f174012a